fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAddonsManagerRobot.kt

```diff
@@ -293,5 +297,18 @@ fun addonsMenu(interact: SettingsSubMenuAddonsManagerRobot.() -> Unit): Settings
     return SettingsSubMenuAddonsManagerRobot.Transition()
 }
 
+private fun addonItem(addonName: String) =
+    onView(
+        allOf(
+            withId(R.id.add_on_item),
+            hasDescendant(
+                allOf(
+                    withId(R.id.add_on_name),
+                    withText(addonName),
+                ),
+            ),
+        ),
+    ).check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
```

Comment:
Are you sure you want this check here, or was it copied over? It might cause failures if the item is not yet visible (you need to scroll to it for example).

Reply:
Yes, I've copied it from [openDetailedMenuForAddon](https://github.com/mozilla-mobile/firefox-android/pull/5628/files#diff-8e8c3790ab858c3fe796c7b006f564e8ebdcf6bb9e337f8d8e2365450a1b1f4eL186) for better readability.

As you can see the scroll to the add-on is the [first step](https://github.com/mozilla-mobile/firefox-android/pull/5628/files#diff-8e8c3790ab858c3fe796c7b006f564e8ebdcf6bb9e337f8d8e2365450a1b1f4eL184) in that function.
However, I need to drop [this](https://github.com/mozilla-mobile/firefox-android/pull/5628/files#diff-8e8c3790ab858c3fe796c7b006f564e8ebdcf6bb9e337f8d8e2365450a1b1f4eR311) check